python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -33,6 +33,11 @@ def test_tidy_config(self):
         self.assertEqual("ignored directory './fake/dir' doesn't exist", errors.next()[2])
         self.assertNoMoreErrors(errors)
 
+    def test_wpt_manifest_checks(self):
+        wrong_path = "/wrong/path.ini"
+        errors = tidy.check_manifest_dirs(wrong_path)
+        self.assertEqual("%s manifest file is required but was not found" % wrong_path, errors.next()[2])
```

Comment:
This doesn't actually test the contents of a manifest that does exist. Additionally, we'll want a `self.assertNoMoreErrors(errors)`.

Reply:
Makes sense. Should I check the existing include.ini or create a new one?

Reply:
Create a new one in python/tidy/servo_tidy_tests/ that contains one valid path and one invalid one.

Reply:
@jdm check it out